itests/pom.xml

```diff
@@ -35,11 +35,11 @@
     <module>org.openhab.core.io.rest.core.tests</module>
     <module>org.openhab.core.model.core.tests</module>
     <module>org.openhab.core.model.item.tests</module>
-    <module>org.openhab.core.model.persistence.tests</module>
     <module>org.openhab.core.model.rule.tests</module>
     <module>org.openhab.core.model.script.tests</module>
     <module>org.openhab.core.model.thing.testsupport</module>
     <module>org.openhab.core.model.thing.tests</module>
+    <module>org.openhab.core.persistence.tests</module>
```

Comment:
I saw these tests were moved to another project in #1551.
Does it even require OSGi for running the tests?

Reply:
At least in their current form, yes.

Reply:
I think it is safe to move all the related tests because from my understanding they do not need OSGi.